frontend/tests/e2e_tests/playwright-qemu.config.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Northern.tech AS
```

Comment:
could you verify the copyrights on the files?

Reply:
🤦 a verified copy paste error in here 🙃
The rest should align with when the files were added to git - otoh seeing that the `settings.spec.ts` connection was lost (similar likely with the `layoutAssertions.spec.ts` that in addition has a bigger distance to the `devices.spec.ts` now)...
would you rather have it adjusted to 2025 or to when the content of the file was started (so 2021)?

Reply:
I took the decision Github made - what's considered new here gets a new year 🤷